admin_manual/release_notes.rst

```diff
@@ -12,6 +13,61 @@ Release Notes
+
+- **Added "occ files:scan" repair mode to repair filecache inconsistencies**
+We recommend to use this command when directed to do so in the upgrade process (TODO more info)
+Please check the ownCloud documentation for more information. **LINK (https://github.com/owncloud/documentation/issues/3446)**
+
+- **Detailed mode for "occ security:routes"**
+Administrators can use the output of this command when using a network firewall to check the appropriateness of configured rules or to get assistance when setting up.
+
+- **Added mode of operations to differentiate between single-instance or clustered setup**
+As ownCloud needs to behave differently when operating in a clustered setup we added a config.php option ``'operation.mode' => 'single-instance' or 'clustered-instance'``. 
+Currently the Market App (ownCloud Marketplace integration) does not support clustered setups and can do harm when used for installing or updating apps. The new config setting prevents this and other actions that are undesired in cluster mode.
+**Therefore when operating in a clustered setup it is mandatory to set this option.**
+Please check the ownCloud documentation for more information. **LINK (https://github.com/owncloud/documentation/pull/3506)**
+
+- **Added occ dav:cleanup-chunks command to clean up expired uploads**
+When file uploads are interrupted by any reason already uploaded file parts (chunks) stay on the storage to be able to resume the file upload in a following upload run. Resuming an upload is only possible until the partial upload is expired and deleted, respectively. To cleanup chunks (expire and delete) originating from unfinished uploads administrators can use this newly introduced command. The expiry time default is two days but can be specified as a parameter of this command.
+**It is recommended to configure CRON to execute this background job regularly (it is not included in the regular ownCloud background jobs to give administrators more flexibility in scheduling)**. Please check the ownCloud documentation for more information. **LINK (https://github.com/owncloud/documentation/pull/3522)**
+
+- **Administrators can now exclude files from integrity check in config.php**
+When administrators did intentional changes to the ownCloud code they now have the ability to exclude certain files from the integrity checker.
+Please check "config.sample.php" for the usage of ``'integrity.excluded.files'``. **DOC LINK??**
+
+- **All amazon locations support v4 now - v3 deprecated**
+Need input on relevance of deprecation for existing external storages and on description of this item @tomneedham
+
+- **Modified time value of files is now 64 bits long**
+Note: When upgrading to 10.0.4 migrations may increase update duration dependent on number of files.
```

Comment:
can't this be done without downtime ... we really should think about a strategy for doing this ... probably with a new table that gets renamed after the column has been changed ... this will take hours on large installations

Reply:
not the right place to ask